ee/codegen/src/__test__/helpers/node-data-factories.ts

```diff
@@ -1075,9 +1075,10 @@ export function apiNodeFactory({
           value: {
             rules: [
               {
-                type: "INPUT_VARIABLE",
+                type: "CONSTANT_VALUE",
```

Comment:
Removed these so that tests don't need input variable contexts defined for unrelated attributes